components/Brand.vue

```diff
@@ -1,8 +1,17 @@
 <template>
 	<div class="brand">
 		<nuxt-link to="/">
-			<img src="~/static/img/We Count logos_colour and bw-01.png" alt="Logo for Project We Count">
+			<WeCountLogo />
 			We Count
 		</nuxt-link>
 	</div>
 </template>
+<script>
+import WeCountLogo from "~/assets/images/logo.svg?inline"
```

Comment:
Was wondering why you implemented the image this way? as opposed to an img tag or another method

Reply:
If you embed and `<svg>` element, you can style parts of it with CSS. That's what enables the colour changes when the UIO+ theme is changed. SVGs in `<img>` tags can't be styled.

Reply:
Ref:

- https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorial/SVG_and_CSS
- https://css-tricks.com/using-svg/

Reply:
Interesting! thanks

Reply:
If there is an aversion to inlining the SVG, you could look at using `<use>`. I wrote up some research on SVGs for icons and styling them, which you can read on the wiki ( https://wiki.fluidproject.org/display/fluid/Research+SVG+Icons ). That was a while ago, so things may have improved since then. And this will also depend on the browser support we want for the site. Although some of our styling choices for the layout may already be dictating that.

Reply:
Okay, will check it out